vue-begin/src/APIService.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+const API_URL = 'http://dummy.restapiexample.com/api/v1';
+
+  export const getEmployees = async () => {
+    const url = `${API_URL}/employees`;
+    const response = await axios.get(url);
```

Comment:
You can destructure response like:
`const { data } = await axios.get(url)
return data
`

Reply:
i createEmployee I can also destructure like in your example?

Reply:
ok